modules/chat/public/views/pages/api/get_messages.json.liquid

```diff
@@ -0,0 +1,40 @@
+---
+slug: api/chat/get_messages
+method: get
+---
+
+{% comment %}
+  Get messages for given conversation
+
+  Params:
+  - conversation id (string, required)
+  - page number for pagination (int, default: 1)
+  - items per page (int, default: 30)
+{% endcomment %}
+
+{% liquid
+  assign current_user_id = context.current_user.id
+
+  assign conversation_id = context.params.conversation_id
+  assign page = context.params.page | plus: 0 | default: 1
+  assign per_page = context.params.per_page | plus: 0 | default: 30
+
+  graphql messages = 'modules/chat/find_messages', conversation_id: conversation_id, page: page, per_page: per_page
+%}
+
+{% if messages.models.results[0] %}
```

Comment:
This should go to `theme/api`

Reply:
Ok, so I've changed the whole structure of the API tree to match that from the main app. Good or no good?